src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlConnectionString.cs

```diff
@@ -1117,8 +1112,7 @@ internal bool ConvertValueToEncrypt()
         {
             // If the Authentication keyword is provided, default to Encrypt=true;
             // otherwise keep old default for backwards compatibility
```

Comment:
The comment here is not right now(after #1210, 4.0.0-preview1), now `defaultEncryptValue` is always true